README.md

```diff
@@ -15,16 +15,20 @@ Then install the framework into the `lib/` directory using:
 
 ```
 mkdir lib/
-pip install -t lib/ https://github.com/canonical/operator
+pip3 install -t lib/ https://github.com/canonical/operator/archive/master.zip
 ```
 
+> Note: Due to [pip#3826](https://github.com/pypa/pip/issues/3826), you may get
+> a "can't combine user with prefix" if you are using pip3 provided by Ubuntu
+> prior to 19.04, in which case you simply need to add `--system` to the command.
+
 Your `src/charm.py` is the entry point for your charm logic. At a minimum, it
 needs to define a subclass of `CharmBase` and pass that into the framework's
 `main` function:
 
 ```python
 import sys
-sys.path.insert.append('lib')
+sys.path.append('lib')
```

Comment:
insert.append does seem a bit strange :).

Reply:
Had just noticed that as well.  Now fixed.